File: Java/1480.java
class Solution {
    public int[] runningSum(int[] nums) {

        int[] new_array = new int[nums.length]; 

        int sum = 0; 

        for(int i = 0; i < new_array.length; i++ ){

            sum += nums[i]; 
            new_array[i] = sum; 

        }

        return new_array; 
        
    }
}

File: Rust/1480.rs
impl Solution {
    pub fn running_sum(nums: Vec<i32>) -> Vec<i32> {

        let mut new_array: Vec<i32> = vec![0; nums.len()];        
        let mut sum: i32 = 0; 

        for i in 0..new_array.len()-1{
             sum += nums[i]
             new_array[i] = sum

        }

        new_array
    }
}